.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,21 @@
+<!-- Thank you for contributing to Rook! -->
```

Comment:
This template needs to be customized for odf, instead of having all the rook references

Reply:
I removed this commit also it doesn't make sense with commitlint commit to me.